code/main.py

```diff
@@ -0,0 +1,285 @@
+        #################################################
+
+        lbl = tk.Label(self.labelFrame, text="Input file")
+        lbl.grid(column=0, row=0, padx=20)
+        self.filename_input = ""
+        self.button_browse_input()
+
+        #################################################
+        ##       RADIO BUTTON - FEATURIZATION TYPE     ##
+        #################################################
+        MODES = [
+            ('privative', 1),
+            ('complementary', 2),
+            ('inferential_complementary', 3),
+            ('full', 4)
+        ]
+        self.featurization = tk.StringVar(None, 'complementary')
+
+        for text, value in MODES:
+            tk.Radiobutton(self.labelFrame3,
+                        indicatoron=0,
+                        text=text,
+                        variable=self.featurization,
+                        value=text).pack(anchor=tk.W, fill=tk.X)
+
+        self.specification = Featurizer.FEATURIZATION_MAP.get(self.featurization.get(), self.featurization.get())
+
+        #################################################
+        ##               CHECK BOX - OPTIONS           ##
+        #################################################
+
+        self.use_numpy = tk.BooleanVar()
+        tk.Checkbutton(self.labelFrame4,
+                    text="numpy",
+                    variable=self.use_numpy).pack(anchor=tk.W, fill=tk.X)
+
+        self.verbose = tk.BooleanVar()
+        tk.Checkbutton(self.labelFrame4,
+                    text="verbose",
+                    variable=self.verbose).pack(anchor=tk.W, fill=tk.X)
+
+        #################################################
+        ##               OUTPUT BUTTONS                ##
+        #################################################
+
+        lbl2 = tk.Label(self.labelFrame5, text="Output file")
+        lbl2.grid(column=0, row=0, padx=20, pady=10)
+
+        self.button_browse_output_GV_feat()
+        self.button_browse_output_PNG_feat()
+        self.button_browse_output_GV_poset()
+        self.button_browse_output_PNG_poset()
+        self.button_browse_output_CSV()
+
+        self.button_print()
+
+    def init_featurizer(self):
+        '''
+            Initialize Featurizer object based on the user selected data
+
+            Note:
+                Selection required for: filename_input, filename_output, featurization type
+        '''
+        featurizer = Featurizer.Featurizer.from_file(
+            self.filename_input, self.specification, use_numpy=self.use_numpy.get(),
+            verbose=self.verbose.get()
+        )
+        verbose_text = featurizer.get_features_from_classes_helper()
+        return featurizer, verbose_text
+
+    def button_browse_input(self):
+        ''' Browse Button for filename_input '''
+        button = ttk.Button(self.labelFrame, text="Browse", command=self.fileDialog)
+        button.grid(column=1, row=0)
+
+    def fileDialog(self):
```

Comment:
Can you switch this method to snake case too? `file_dialog`. Ditto with the other fileDialog methods below.